mne_bids/write.py

```diff
@@ -827,7 +826,8 @@ def write_raw_bids(raw, bids_path, bids_root, events_data=None,
     bids_path : str | BIDSPath
         The base filename of the BIDS compatible files. Typically, this can be
         generated using :func:`mne_bids.BIDSPath`, and then calling
-        ``basename`` property.
+        ``basename`` property. The path to root of the
+        BIDS folder must be passed in via the ``BIDSPath`` object.
```

Comment:
How about
> The file to write. The :class:`mne_bids.BIDSPath` instance passed here **must**
> have the ``.root`` and ``.modality`` attributes set.
not sure about `.modality` though! Is `.suffix` required too? and `.extension`?

Reply:
Suffix and extension aren't needed because those get automatically inferred by write_raw_bids